CONTRIBUTING.md

```diff
@@ -74,6 +77,22 @@ Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-
 - Dates in a release are approximations and always subject to change.
 - `Next` milestone is for work that has been triaged, but not prioritized/accepted for any release.
 
+## Proposal process (CAEP)
+
+The Cluster API Enhacement Proposal is the process this project uses to adopt new features, or changes to the APIs.
```

Comment:
Might be add that a CAEP could be required also for changes without impacts on the API (e.g cluterctl, clusterctl templating)

Reply:
Along the lines of @fabriziopandini's comment:
```suggestion
The Cluster API Enhacement Proposal is the process this project uses to adopt new features, changes to the APIs, changes to contracts between components, or changes to CLI interfaces.
```